Packs/CrowdStrikeFalcon/TestPlaybooks/Test_Playbook_-_CrowdStrike_Falcon_-_Get_Detections_by_Incident.yml

```diff
@@ -0,0 +1,3738 @@
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 2940,
+          "y": -2280
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "360":
+    id: "360"
+    taskid: c93d06e0-ce15-4341-8155-5f1c7c8ebc41
+    type: title
+    task:
+      id: c93d06e0-ce15-4341-8155-5f1c7c8ebc41
+      version: -1
+      name: Done verifying Incident Fields
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "202"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 5360,
+          "y": -2310
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "361":
+    id: "361"
+    taskid: 7aba1e67-9dcb-499a-8382-e9890d2b9d32
+    type: title
+    task:
+      id: 7aba1e67-9dcb-499a-8382-e9890d2b9d32
+      version: -1
+      name: Start Testing
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "339"
+      - "255"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -400,
+          "y": -2780
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "362":
+    id: "362"
+    taskid: 46570c09-ac84-4b5a-861b-06f4b463d2db
+    type: playbook
+    task:
+      id: 46570c09-ac84-4b5a-861b-06f4b463d2db
+      version: -1
+      name: CrowdStrike Falcon - Get Detections by Incident
+      description: |-
+        This playbook is part of the 'Malware Investigation And Response' pack. For more information, refer to https://xsoar.pan.dev/docs/reference/packs/malware-investigation-and-response.
+        This playbook enables getting CrowdStrike Falcon detection details based on the CrowdStrike incident ID.
+      playbookName: CrowdStrike Falcon - Get Detections by Incident
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "243"
+    scriptarguments:
+      IncidentID:
+        simple: ldt:046761c46ec84f40b27b6f79ce7cd32c:223338720900
```

Comment:
we need to avoid such use cause this incident can be rotated somewhere in the nearby future ... Can we check for something similar with the list incident command?

Reply:
Added the following tasks:

1- 'Get CrowdStrike Falcon Available Detections' task (task number 366) - the task lists available detections.

2- 'Check CrowdStrike Falcon Available Incidents' - task (task number 367) - the task tests if there are available detections for testing processes.

2- 'Print Error Message - No detections found' task (task number 368) - the task prints an error message if no CrowdStrike Falcon available detections were found.